LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/color-scheme-dependent-color-stops.html

```diff
@@ -48,7 +49,7 @@
 <div id="light-dark-in-color-mix" class="box"></div>
 <script>
 
-setTimeout(() => {
+step_timeout(() => {
```

Comment:
No strong preference, but does rAF work? I _think_ importing `testharness.js` can be troublesome in reftests can be troublesome as it renders extra output but maybe I'm wrong? Can you export to WPT and check the cross browser results to see how WebDriver runs it?

Reply:
Looks like it does indeed cause a harness error to include `testharness.js` https://github.com/web-platform-tests/wpt/pull/45463/checks?check_run_id=23330003714

Reply:
Updated to use rAF.